boost-maven/boost-boms/pom.xml

```diff
@@ -14,7 +14,6 @@
         <version>0.1.3-SNAPSHOT</version>
     </parent>
 	
-	<groupId>io.openliberty.boost</groupId>
```

Comment:
Just curious - is this removed because the group id is inherited from the parent?

Reply:
Yep, and the version is as well.